devlab/provision/generate_config.sh

```diff
@@ -34,9 +34,7 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    if [[ -n ${value} ]]; then
-      sed -i "s|<${key}>|${value}|g" ${result_config}
-    fi
+    sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
Looks like after this operation we will have key without value in the configurationfile. I think it is possible to remove key line without value. Like this:
    if [[ -n ${value} ]]; then
        sed -i "s|<${key}>|${value}|g" ${result_config}
    elif [[ -z ${value} && -n ${key} ]]; then
        sed -i "/$key/d" ${result_config}
    fi